client/src/components/BigPortfolioWarning.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { t, Trans } from "@lingui/macro";
+import { withI18n, withI18nProps } from "@lingui/react";
+import warning from "../assets/img/icon-warning.svg";
+
+export const PORTFOLIO_SIZE_THRESHOLD = 300;
+
+type BigPortfolioWarningProps = withI18nProps & {
+  size: number;
```

Comment:
`size` sounds a little ambiguous. I'd make this type a bit more descriptive, like `sizeOfPortfolio` or something along those lines.